src/java/TypesUtils.cpp

```diff
@@ -342,6 +342,7 @@ static bool isJavaMap(JNIEnv * env, jobject obj)
 
 static Object * hashmapJavaToObjectConverter(JNIEnv * env, jobject obj)
 {
+	env->PushLocalFrame(20);
```

Comment:
What does 20 or 10 means? is it going to be the limit?
If that's just a hint and not the limit, could you use the same value everywhere (and maybe use a constant)?